bchrpc/server.go

```diff
@@ -929,6 +975,15 @@ func (s *GrpcServer) GetAddressUnspentOutputs(ctx context.Context, req *pb.GetAd
 				continue
 			}
 
+			var slpToken *pb.SlpToken
+			if s.slpIndex != nil {
+				slpToken, _ = s.getSlpToken(&txHash, uint32(i))
```

Comment:
For this particular case, if an error happens on line 980 then the condition on the following line will be skipped.  So I'm thinking it would be acceptable to leave this one as is.